src/woo_publications/publications/tests/test_publications_admin.py

```diff
@@ -244,15 +251,173 @@ def test_publications_admin_create(self):
             self.assertEqual(
                 str(added_item.laatst_gewijzigd_datum), "2024-09-25 00:14:00+00:00"
             )
-            self.assertEqual(
-                added_item.bron_bewaartermijn, "Selectielijst gemeenten 2020"
-            )
-            self.assertEqual(added_item.selectiecategorie, "20.1.2")
+            # Test if the fields get overwritten from the input by the IC
+            self.assertEqual(added_item.bron_bewaartermijn, "bewaartermijn")
+            self.assertEqual(added_item.selectiecategorie, "selectiecategorie")
             self.assertEqual(
                 added_item.archiefnominatie, ArchiveNominationChoices.retain
             )
-            self.assertEqual(str(added_item.archiefactiedatum), "2025-01-01")
-            self.assertEqual(added_item.toelichting_bewaartermijn, "extra data")
+            self.assertEqual(str(added_item.archiefactiedatum), "2029-09-25")
+            self.assertEqual(added_item.toelichting_bewaartermijn, "toelichting")
+
+    def test_publication_admin_create_retention_field_with_both_archive_nomination_choices(
+        self,
+    ):
```

Comment:
These tests would be much better suited as model unit tests with the suggested method above - you have less setup to do and the assertions become simpler.

Reply:
I kept these old test because we need to have some test proving that it is applied in the admin, but I also added model tests for this.

Reply:
One minimal "create" test that proves admin actions calculate the archiving details are sufficient. On top of that, you want one "update" test where the archiving parameters are overridden, and assert that the model method is not invoked again. All the rest can go. And with minimal I also mean: just a single IC is enough.

All these extra tests have no added value over the model tests, but they will cause problems when behaviour changes and we have to update them. It's the testing pyramid:

1. Have unit tests (the model method) for the various edge cases
2. Have an integration tests proving that the model method is invoked when it should (the admin test)

Reply:
I won't write the test for the update, because this will be done in #201. Especially because the logic for when it does and doesn't need to be invoked will change.